Source/Core/VideoCommon/OnScreenDisplay.cpp

```diff
@@ -56,7 +61,7 @@ void DrawMessages()
 
 		alpha <<= 24;
 
-		g_renderer->RenderText(it->str, left, top, 0xffff30 | alpha);
+		g_renderer->RenderText(it->m_str, left, top, it->m_rgb | alpha);
```

Comment:
Fancy feature: Let's move the masking here and multiply the alpha value, so it's allowed to set an alpha value per message.